cvmfs/swissknife_lease_json.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef CVMFS_SWISSKNIFE_LEASE_JSON_H_
+#define CVMFS_SWISSKNIFE_LEASE_JSON_H_
+
+#include "swissknife_lease_curl.h"
+
+#include <string>
+
+bool ParseAcquireReply(const CurlBuffer& buffer, std::string* session_token);
```

Comment:
Maybe overkill maybe not: we might want to encapsulate parsing the JSON objects in a class.